### DISCUSSION TITLE
Manual build of ghidra using Debian

### DISCUSSION THREAD

**Discussion by graps01** [NationalSecurityAgency/ghidra#5086] [Q&A]
Hi All,

While there is already a binary available for Debian, I thought it would be interesting to do a full build of the ghidra project using Debian (bullseye). I have learned a lot, and am still learning about the build, from a command line usage and perspective.

For instance, in the "gradle buildGhidra" instruction, what is the library or initialization string/setting for building ghidra with Debian ? I know that I can use the --debug option to analyze the build progress, but do not have a solid grasp of the init strings. For now, I am studying up gradle init strings and build settings.

**Comment by ryanmkurtz** [comment]
Building on Debian is no different than any other OS.  It's just the couple of steps outlined in the [README.md](https://github.com/NationalSecurityAgency/ghidra/blob/master/README.md) file.
